lib/abbreviato/abbreviato.rb

```diff
@@ -22,6 +22,6 @@ def self.truncate(source = '', user_options = {})
     truncated_sax_document = TruncatedSaxDocument.new(DEFAULT_OPTIONS.merge(user_options))
     parser = Nokogiri::HTML::SAX::Parser.new(truncated_sax_document)
     parser.parse(source) { |context| context.replace_entities = false }
-    [truncated_sax_document.truncated_string, truncated_sax_document.truncated]
+    [truncated_sax_document.truncated_string.strip, truncated_sax_document.truncated]
```

Comment:
Looks like the parser does not strip trailing spaces so I had to update this to fix the spec test
```
expected: ""
            got: "  "
       (compared using ==)
     # ./spec/support/spec_helpers/abbreviato_macros.rb:6:in `block in test_truncation'
```